modules/client/test/integration/client/methods.test.ts

```diff
@@ -1691,5 +1695,284 @@ describe("Client", () => {
       //   "Invalid ENS name"
       // );
     });
+
+    describe("isDaoUpdateProposalValid", () => {
+      it("should check if a proposal is valid for updating the dao and fail with an invalid proposal id", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new Client(ctx);
+        const proposalId = TEST_DAO_ADDRESS;
+        expect(
+          () => client.methods.isDaoUpdateProposalValid(proposalId),
+        ).rejects.toThrow(new InvalidProposalIdError());
+      });
+
+      it("should check if a proposal is valid for updating the dao and fail with a proposal that does not exist", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new Client(ctx);
+        const proposalId = TEST_MULTISIG_PROPOSAL_ID;
+        const mockedClient = mockedGraphqlRequest.getMockedInstance(
+          client.graphql.getClient(),
+        );
+        mockedClient.request.mockResolvedValueOnce({
+          iplugin: null,
+        });
+        expect(
+          () => client.methods.isDaoUpdateProposalValid(proposalId),
+        ).rejects.toThrow(new ProposalNotFoundError());
+      });
+      it("should check if a proposal is valid for updating the dao and fail with a proposal that has more than one action", async () => {
```

Comment:
are these restrictions also in place in the frontend? because a proposal can not only upgrade DAO but also install a plugin at the same time. would that kind of proposal return a false in the check if the proposal is valid?

Reply:
```suggestion
      it("invalidates a proposal that has more than one action and states the invalidity cause", async () => {
```
Let's be more precise in writing the tested behavior in the test description.

Reply:
After talking with james today this tests dont make sene anymore, so i'll need to mupdate them